nrf_security/configs/nrf-config.h.template

```diff
@@ -1644,7 +1644,18 @@
  *
  * Comment this macro to disable support for TLS 1.2 / DTLS 1.2
  */
-#define MBEDTLS_SSL_PROTO_TLS1_2
+#cmakedefine MBEDTLS_SSL_PROTO_TLS1_2
+
+/**
+ * \def MBEDTLS_SSL_SOME_SUITES_USE_TLS_CBC
+ *
+ * Enable the CBC construct used in TLS 1.2
+ *
+ * Requires: MBEDTLS_SSL_PROTO_TLS1_2
+ *
+ * Comment this macro to disable the CBC contrsuct for TLS 1.2
+ */
+#cmakedefine MBEDTLS_SSL_SOME_SUITES_USE_TLS_CBC
```

Comment:
This seem to be Nordic addition, so should probably be moved to the bottom of this file and keep the previous comment:
```
* Nordic added.
```